apps/docs/src/docs/components/demo/PopoverOverview.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <!-- #region template -->
+  <div class="d-flex gap-2">
+    <BButton v-b-popover.focus.top="'I am popover directive content!'" title="Popover Title">
+      Hover Me
+    </BButton>
+
+    <BButton id="popover-target-1"> Hover Me </BButton>
+    <BPopover target="popover-target-1" placement="top">
+      <template #title>Popover Title</template>
+      I am popover <b>component</b> content!
+    </BPopover>
```

Comment:
```suggestion
    <BPopover>
      <template #target> <BButton> Hover Me </BButton></template>
      <template #title>Popover Title</template>
      I am popover <b>component</b> content!
    </BPopover>
```

I'd prefer we suggest the slot usage over having to put id:s and targets on elements, less errorprone.